packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/index.md

```diff
@@ -4,13 +4,28 @@ excerpt: How to implement authorization flows by grant type with Okta
 layout: Guides
 ---
 
-<ApiAmProdWarning />
-
-## Overview
+<StackSelector />
 
 This document guides you through implementing an OAuth 2.0 authorization flow for your application by grant type with Okta.
 
-Select the authorization grant-type flow to implement: <StackSelector />
+---
+
+**Learning outcomes**
+
+* Understand OAuth 2.0 grant-type flows.
+* Learn how to set up your app for specific grant types.
+* Learn how to implement the grant-type flows.
```

Comment:
```suggestion
* Implement the grant-type flows.
```

Reply:
@vanngo-okta, I had put "Learn..." in my nut facts originally. However, Chris indicated that he doesn't think it's necessary and we should just say it directly, for example, "Implement...", "Set up...". I like having the "Learn..." there, and may just put it back in when I evaluate and do the overall clean-up.